stable/insights-agent/templates/opa/annotation-blocked.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.opa.enabled -}}
+apiVersion: insights.fairwinds.com/v1
```

Comment:
I like that you built a handful of test checks!

We should probably provide these to folks in another context, rather than automatically including them with an OPA install. We're anticipating OPA users are mostly going to want to do their own thing, but it'd be good to provide a handful of examples and pre-built checks. Let's keep them in our pocket for now.